lib/honeydew/node_monitor_supervisor.ex

```diff
@@ -1,10 +1,15 @@
 defmodule Honeydew.NodeMonitorSupervisor do
   alias Honeydew.NodeMonitor
 
-  def start_link(queue, nodes) do
-    import Supervisor.Spec
+  def child_spec(opts) do
+    %{
+      id: __MODULE__,
```

Comment:
honeydew isn't an app that's started under the application controller on its own, so it doesn't have its own supervision tree. other apps ask honeydew for a supervision spec, based on the queue name, so they're all independent. if we register this name globally, we'd only be able to run one queue in a VM.